src/Tools/Github/GitMergeBot/Program.cs

```diff
@@ -83,7 +83,13 @@ private Program(Options options)
             }
 
             var newBranchName = await MakePrBranch(_options.SourceUser, _options.RepoName, remoteIntoBranch, newBranchPrefix);
-            await SubmitPullRequest(newBranchName);
+            var pullRequest = await SubmitPullRequest(newBranchName);
+
+            // pullRequest could be null if we are running in debug mode.
+            if (pullRequest != null)
+            {
+                await CommentOnIssue(_options.SourceUser, _options.RepoName, pullRequest.Number, "@dotnet-bot test vsi please");
```

Comment:
I vote for keeping the "@dotnet-bot". @davkean (I think) mentioned something about this w.r.t. StashPop, which is why the default options for retesting inside the `dotnet` org include the unnecessary-but-clarifying "@dotnet-bot"